scripts/vcf_site_level_filtering_cromwell_tests/README.md

```diff
@@ -0,0 +1,9 @@
+# Filtering Automated Tests for WDL
+
+**This directory is for GATK devs only**
+
+This directory contains scripts for running Variant Site Level WDL tests in the automated travis build environment.
```

Comment:
Is this Travis reference obsolete?

Reply:
I think so.  Inasmuch as this is just in a document Sam put together to document his testing, I am less inclined to edit out poor wording.  We yanked his tests into this PR, but I'd only want to go in an actively edit his stuff if we absolutely need to.